src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigCommandExecutor.java

```diff
@@ -31,6 +31,7 @@ protected void addBaseArgumentsLine(List<String> commandsList) {
             addArgumentKeyVal("tokenfile", baseArgs.getTokenFile(), commandsList);
         }
         addArgumentKeyVal("appname", baseArgs.getAppName(), commandsList);
+        addArgumentKeyVal("proxy", baseArgs.getProxy(), commandsList);

```

Comment:
Looks good.
Please see that:
1. We are not missing any other fields.
2. Proxy is working on other commands as well.
3. We are not missing any other fields the rest of the commands.

Reply:
1. A PR-BSID is created - it means it works well.
2. Tested - seems that it works well
3. I cant tell, you want me to do a full QA cycle? this plugin is running for while and we did not hear a lot of missing fields